packages/client/src/components/Router.tsx

```diff
@@ -1,13 +1,13 @@
 import React, { Suspense } from "react";
-import { BrowserRouter, Redirect, Route, Switch } from "react-router-dom";
+import { Redirect, Route, Switch } from "react-router-dom";
 
 import LoadingPage from "../pages/LoadingPage";
 import { redirectConfig, routeConfig } from "../routes";
 import ScrollToTop from "./ScrollToTop";
 
 const Router = () => {
   return (
-    <BrowserRouter>
+    <>
```

Comment:
Maybe add a comment that browser router is actually defined somewhere else. This could be confusing.